scripts/deploy/deploy.js

```diff
@@ -39,30 +39,48 @@ const uploadPathTypes = process.argv[4].split(',')
 runMain(async () => {
   const awsConfig = AWS_CONFIG[env]
   let cloudfrontPathsToInvalidate = []
-  for (const { packageName } of packages) {
-    const bundleFolder = buildBundleFolder(packageName)
-    for (const uploadPathType of uploadPathTypes) {
-      let uploadPath
-      if (uploadPathType === 'pull-request') {
-        const pr = await fetchPR(LOCAL_BRANCH)
-        if (!pr) {
-          console.log('No pull requests found for the branch')
-          return
-        }
-        uploadPath = buildPullRequestUploadPath(packageName, pr.number)
-      } else if (uploadPathType === 'root') {
-        uploadPath = buildRootUploadPath(packageName, version)
-      } else {
-        uploadPath = buildDatacenterUploadPath(uploadPathType, packageName, version)
-      }
-      const bundlePath = `${bundleFolder}/${buildBundleFileName(packageName)}`
+  for (const { packageName, chunks } of packages) {
+    const pathsToInvalidate = await uploadPackage(awsConfig, packageName, chunks)
+    cloudfrontPathsToInvalidate.push(...pathsToInvalidate)
+  }
+  invalidateCloudfront(awsConfig, cloudfrontPathsToInvalidate)
+})
+
+async function uploadPackage(awsConfig, packageName, chunks) {
+  const cloudfrontPathsToInvalidate = []
+  const bundleFolder = buildBundleFolder(packageName)
+
+  for (const uploadPathType of uploadPathTypes) {
+    for (const chunkName of chunks) {
```

Comment:
**💬 suggestion:** ‏Instead of hardcoding the chunks here, we could use `fs.readdirSync` and upload all files in the `bundles/` folder.

Reply:
Good idea! Done.